src/index.ts

```diff
@@ -0,0 +1,49 @@
+import * as mtr from 'mongo-to-rabbit';
+import express from 'express';
+import * as http from 'http';
+import { router } from './router';
+import { colToQueueArray, mongoConnectionString, rabbitURI, port } from './config';
+import { MongoDataType, RabbitDataType } from 'mongo-to-rabbit/lib/paramTypes';
+import { log, Severity } from './logger';
+
+function initApp() {
+  const app : express.Application = express();
+  app.use(router);
+  app.set('port', port);
+  http.createServer(app).listen(app.get('port'), () => {
+    log(Severity.INFO, `Express server listening on port ${app.get('port')}`, 'initApp');
+  });
+}
+
+async function initWatchAndNotify() : Promise<void> {
+  for (const colQCouple of colToQueueArray) {
```

Comment:
there is no reason for it to not be const, but I will change it to forEach so that it will be more readable.

Reply:
I needed an async loop. `forEach `can't provide that as far as I know, so I used this instead.
There is no problem with the `const` since the loop recreates the variable in every iteration.

Reply:
walla